### PR TITLE
Post class introduced in "mod" and "include"

### PR DESCRIPTION
**Pull request by annando**
This PR introduced the `Post` class in all files that aren't a class yet.

Edit: It seems as if this PR together with PR https://github.com/friendica/friendica/pull/9800 should be accepted first. The the other one can follow.

### CODE REVIEW

File: include/conversation.php [hunk 1]
```diff
@@ -33,10 +33,11 @@
 use Friendica\DI;
 use Friendica\Model\Contact;
 use Friendica\Model\Item;
+use Friendica\Model\Post;
 use Friendica\Model\Profile;
 use Friendica\Model\Tag;
 use Friendica\Model\Verb;
-use Friendica\Object\Post;
+use Friendica\Object\Post as ObjectPost;
```
Comment (MrPetovan) [inline include/conversation.php]: More natural in English.
```suggestion
use Friendica\Object\Post as PostObject;
```

File: include/conversation.php [hunk 2]
```diff
@@ -677,7 +678,7 @@ function conversation(App $a, array $items, $mode, $update, $preview = false, $o
 				$item['pagedrop'] = $page_dropping;
 
 				if ($item['gravity'] == GRAVITY_PARENT) {
-					$item_object = new Post($item);
+					$item_object = new ObjectPost($item);
```
Comment (MrPetovan) [inline include/conversation.php]:
```suggestion
					$item_object = new PostObject($item);
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Review by MrPetovan — APPROVED** [review]

File: include/conversation.php (hunk 1 above)
Comment (annando) [reply include/conversation.php]: That had been done automatically by my IDE :-)

Comment (MrPetovan) [reply include/conversation.php]: It makes sense with the order of the namespace but not in natural English.

**Pull request merged**